com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -938,6 +938,29 @@ SkinnedMeshRenderer unitySkin
 
                 // add bind pose
                 ExportBindPose (unitySkin, fbxNode, fbxScene, skinnedMeshToBonesMap);
+
+                // now that the skin and bindpose are set, make sure that each of the bones
+                // is set to its original position
+                var bones = unitySkin.bones;
+                foreach (var bone in bones)
+                {
+                    var fbxBone = MapUnityObjectToFbxNode[bone.gameObject];
+                    ExportTransform(bone, fbxBone, newCenter: Vector3.zero, TransformExportType.Local);
+
+                    // cancel out the pre-rotation from the exported rotation
+                    var fbxPreRotationEuler = fbxBone.GetPreRotation(FbxNode.EPivotSet.eSourcePivot);
+                    // Get the inverse of the prerotation
+                    var fbxPreRotationInverse = ModelExporter.EulerToQuaternion(fbxPreRotationEuler);
+                    fbxPreRotationInverse.Inverse();
```

Comment:
nit: That's a bit confusing. Maybe comment 1: "get the pre-rotation...", then comment 2: " ... then reverse it"
As it is now, I'm not sure if we get the inverse or invert the inverse